play-json/src/test/scala/com/lightbend/lagom/scaladsl/playjson/PlayJsonSerializerSpec.scala

```diff
@@ -118,6 +118,23 @@ class PlayJsonSerializerSpec extends WordSpec with Matchers {
       }
     }
 
+    "pick up serializers for registry with migration" in withActorSystem(TestRegistry3) { system =>
+
+      val migratedEvent = MigratedEvent(addedField = 2, newName = "some value")
+
+      val serializeExt = SerializationExtension(system)
+      val serializer = serializeExt.findSerializerFor(migratedEvent).asInstanceOf[SerializerWithStringManifest]
+
+      val bytes = serializer.toBinary(migratedEvent)
+      val manifest = serializer.manifest(migratedEvent)
+
+      bytes.isEmpty should be(false)
+
+      val deserialized = serializer.fromBinary(bytes, manifest)
```

Comment:
one additional behavior in `fromBinary` is to allow the version of deseiralization target type to be the same as that defined in migration. So an earlier target type version 1 will remain unaffected. 
I added an assertion of this manifest just as how it is built manually in `c06c53a`